site/en/install/source_windows.md

```diff
@@ -256,12 +282,12 @@ considered a Unix absolute path since it starts with a slash.)
 
 Add the Bazel and Python installation directories to your `$PATH` environmental
 variable. If Bazel is installed to `C:\tools\bazel.exe`, and Python to
-`C:\Python36\python.exe`, set your `PATH` with:
+`C:\Python310\python.exe`, set your `PATH` with:
```

Comment:
Is there a way to find out this path so the commands don't change depending on the python version?